addons/stock/models/stock_move.py

```diff
@@ -212,6 +212,7 @@ def _compute_location_dest_id(self):
     def _compute_display_assign_serial(self):
         for move in self:
             move.display_import_lot = (
+                move.product_id and
```

Comment:
I'd add `product_id` in the `depends` as well. I know it's implicit through the `has_tracking`, but it doesn't hurt.